camayoc/tests/qcs/api/v1/scans/test_multi_source_scans.py

```diff
@@ -0,0 +1,48 @@
+# coding=utf-8
+"""Tests for ``Scan`` API endpoint for quipucords server.
+
+:caseautomation: automated
+:casecomponent: api
+:caseimportance: high
+:caselevel: integration
+:requirement: Sonar
+:testtype: functional
+:upstream: yes
+"""
+
+import pytest
+
+from camayoc.tests.qcs.api.v1.utils import (
+    prep_all_source_scan,
+    wait_until_state,
+)
+
+
+@pytest.mark.parametrize('scan_type', ['connect', 'inspect'])
+def test_multi_source_create(shared_client, cleanup, scan_type):
+    """Run a scan on a collection of sources and confirm it completes.
+
+    :id: 43624cc1-6c41-4c2d-9919-c3d0aae83165
+    :description: Create sources for each source specified in the
+        config file and then create a scan that scans all the sources.
+        Provided the resources are specified in the config file, this will
```

Comment:
Did you mean `Provided resources`? (s/the resources/resources)

Reply:
I think this is legal, but maybe not common in all regional dialects. https://dictionary.cambridge.org/us/dictionary/english/provided-that

Reply:
Thanks for the clarification, that sounded strange to me but I am not a native english speaker